test/recipes/80-test_ssl_old.t

```diff
@@ -445,6 +443,8 @@ sub testssl {
                 $flag = "-ssl3";
             } elsif ($protocol eq "TLSv1.2") {
                 $flag = "-tls1_2";
+            } elsif ($protocol eq "TLSv1.3") {
```

Comment:
Did you try this with enable-tls1_3? Because I don't think that ssltest_old.c actually understands TLS1.3.